src/main/java/io/ebeaninternal/server/transaction/TransactionManager.java

```diff
@@ -571,9 +579,8 @@ public void visitMetrics(MetricVisitor visitor) {
    * Begin an implicit transaction.
    */
   public SpiTransaction beginServerTransaction() {
-    SpiTransaction t = createTransaction(false, -1);
-    scopeManager.set(t);
-    return t;
+    SpiTransaction transaction = createTransaction(false, -1);
+    return wrapAsScopedTransaction(transaction);
```

Comment:
To get erverything working, I had to put only scopedTransaction in the DefaultTransactionScopeManager, respectively to the ThreadLocal.
When a non scoped transaction is put to the ThreadLocal, it will not get removed. It will only get inactive, so that it can be replaced the next run.
Do you think the additional overhead we will get here, is a problem?